app/assets/scripts/views/analytics-admin-area.js

```diff
@@ -27,11 +27,16 @@ var AnalyticsAA = React.createClass({
     const done = ids.filter(v => v.inTheDatabase).length;
     const total = ids.length;
     const completion = ((done / total) * 100);
+    const bucket = 'https://s3.amazonaws.com/openroads-vn-dumps/by-province-name/';
+    const file = data.provinceName.match(/ /g) ? data.provinceName.replace(' ', '+') : data.provinceName;
```

Comment:
Spaces are fine in the S3 URLs (I think both space _and_ `+` are accepted when an S3 key has a space in it).